debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotChangeEventSource.java

```diff
@@ -388,6 +451,21 @@ private void createDataEventsForTable() {
         catch (SQLException e) {
             throw new DebeziumException("Snapshotting of table " + currentTable.id() + " failed", e);
         }
+        return true;
+    }
+
+    private boolean checkSchemaChanges(ResultSet rs) throws SQLException {
+        if (!connectorConfig.supportsSchemaChangesDuringIncrementalSnapshot()) {
+            return false;
+        }
+        IncrementalSnapshotSchema schema = new IncrementalSnapshotSchema(rs.getMetaData());
+        if (!schema.equals(context.getSchema())) {
+            context.setSchemaVerified(false);
+            IncrementalSnapshotSchema oldSchema = context.getSchema();
```

Comment:
This variable is not used. Maybe it was intended for logging?

Reply:
Added logging